TSC-Charter.md

```diff
@@ -216,6 +260,39 @@ TSC.
 * **Maintainer**: a Collaborator within a Core Project elected to
 represent the Core Project on the TSC.
 
+## Section 11. Escalation of Disputes and Code of Conduct Violations
+
+Participation in the Node.js project is governed by a Code of Conduct policy
+that is established by the TSC. The TSC is required to establish a policy
+for the enforcement of Code of Conduct issues. Any reports of Code of Conduct
+or policy violations on the part of TSC members that are not resolvable through
+that established policy will be referred to binding, independent third-party
+arbitration with a Foundation appointed mediator. All TSC members, upon
+acceptance of their nomination to the TSC body, voluntarily agree to abide by
+the decisions of the independent third-party mediator.
+
+## Section 12: Leaves of Absence and Temporary Suspension of TSC Membership
+
+TSC Members may voluntarily temporarily suspend their memberships in the TSC
+for any period of time not in excess of twelve months. In such cases, the member
+may return to full TSC membership status at any time.
+
+TSC Members may have their memberships involuntarily suspended by a simple
+majority vote of the other TSC members for any period of time not in excess of
+twelve months. The suspension period must be specified in advance of the
+decision. The members privileges as a TSC member will be restored automatically
+after the given period has passed. The TSC may decide, through simple majority
+vote, to lift the suspension early.
+
+## Section 13: Selection of Mediators
+
+When it becomes necessary within the context of this charter to select a
+third party mediator for arbitration, a qualified mediator shall be selected by
+the Node.js Foundation Board of Directors (or an individual designated by the
```

Comment:
We refer to the Node.js Foundation as providing the mediator, but here we state that the board is responsible for selection. Introducing the board into this statement may be unnecessary. I recommend reviewing this bit with Foundation Legal.